### PR TITLE
'Loading Data Set' message not translated

### PR DESCRIPTION
**Pull request by FransMeerhoff**
- Some actions from the file menu are not translated.
- Solves https://github.com/jasp-stats/INTERNAL-jasp/issues/842
- Audit translation updated.

### CODE REVIEW

File: JASP-Desktop/data/importers/importer.h [hunk 1]
```diff
@@ -8,9 +8,13 @@
 
 class ImportDataSet;
 class ImportColumn;
+#include <QCoreApplication>
+#include "utilities/qutils.h"
+
 
 class Importer
 {
+	Q_DECLARE_TR_FUNCTIONS(CSVImporter)
```
Comment (boutinb) [inline JASP-Desktop/data/importers/importer.h]: Why CSVImporter? Should it not be Impoter?

File: JASP-Desktop/osf/onlinedatanodeosf.cpp [hunk 2]
```diff
@@ -77,17 +77,17 @@ void OnlineDataNodeOSF::nodeInfoReceived() {
...
+		case OnlineDataNodeOSF::NewFile:	progressmsg = tr("Creating new file ");		break;
+		case OnlineDataNodeOSF::NewFolder:	progressmsg = tr("Creating new folder ");	break;
+		default:							progressmsg = tr("Osf action ");				break;
 	}
 
 
 	if (reply->error() != QNetworkReply::NoError)
 	{
-		emit progress(progressmsg + _expectedName + " failed because " + reply->errorString(), 0);
+		emit progress(progressmsg + _expectedName + tr(" failed because ") + reply->errorString(), 0);
```
Comment (boutinb) [inline JASP-Desktop/osf/onlinedatanodeosf.cpp]: Use tr("....%1...%2").arg(..).arg(..) here

File: JASP-Desktop/po/jasp_nl.po [hunk 3]
```diff
@@ -14957,7 +14956,8 @@ msgctxt "Form|"
 msgid ""
 "This analysis was created with an older version of JASP (or a dynamic module)"
 msgstr ""
-"Deze analyse is gemaakt met een oudere versie van JASP (of met een dynamische module)"
+"Deze analyse is gemaakt met een oudere versie van JASP (of met een "
+"dynamische module)"
```
Comment (boutinb) [inline JASP-Desktop/po/jasp_nl.po:14960]: Why break the line in 2 ?

File: JASP-Engine/JASP/po/R-nl.po [hunk 4]
```diff
@@ -615,8 +631,16 @@ msgstr "Geobserveerde Percentages vs. Wet van Benford"
 msgid "Observed"
 msgstr "Geobserveerd"
 
-msgid "<b>Figure %i:</b> The observed percentages of each leading number in the population compared to the expected percentage under Benford's law. The more the blue dots lie near the top of the grey bars, the more the population conforms to Benford's law."
-msgstr "<b>Figuur %i:</b> De geobserveerde percentages van ieder leidend nummer in de populatie vergeleken met het verwachte percentage onder de wet van Benford. Hoe dichter de blauwe stippen rond de bovenkant van de grijze staven liggen, hoe meer de populatie overeenstemt met de wet van Benford."
+msgid ""
+"<b>Figure %i:</b> The observed percentages of each leading number in the "
```
Comment (boutinb) [inline JASP-Engine/JASP/po/R-nl.po:635]: Why a new line here?

File: JASP-Desktop/data/importers/importer.h (hunk 1 above)
Comment (FransMeerhoff) [reply JASP-Desktop/data/importers/importer.h]: Strange, I changed it.

File: JASP-Desktop/osf/onlinedatanodeosf.cpp (hunk 2 above)
Comment (FransMeerhoff) [reply JASP-Desktop/osf/onlinedatanodeosf.cpp]: OK

File: JASP-Desktop/po/jasp_nl.po (hunk 3 above)
Comment (FransMeerhoff) [reply JASP-Desktop/po/jasp_nl.po:14960]: That's what lupdate does. It is (of course) not a line break that is shoewn.

File: JASP-Engine/JASP/po/R-nl.po (hunk 4 above)
Comment (FransMeerhoff) [reply JASP-Engine/JASP/po/R-nl.po:635]: Same issue, translate tooling: 
msgid ""
"<b>Figure %...
Is exactly the same as msgid "<b>Figure %..

File: JASP-Desktop/osf/onlinedatanodeosf.cpp (hunk 2 above)
Comment (FransMeerhoff) [reply JASP-Desktop/osf/onlinedatanodeosf.cpp]: OK

File: JASP-Desktop/po/jasp_nl.po (hunk 3 above)
Comment (FransMeerhoff) [reply JASP-Desktop/po/jasp_nl.po:14960]: lupdate does this (No real line braks of course)

**Review by boutinb — APPROVED** [review]

**Pull request merged**